app/config/FrontendAppConfig.scala

```diff
@@ -36,9 +37,13 @@ class FrontendAppConfig @Inject()(configuration: Configuration) {
   val betaFeedbackUnauthenticatedUrl = s"$contactHost/contact/beta-feedback-unauthenticated"
   val signOutUrl: String             = configuration.get[String]("urls.logout")
 
-  private val declareUnloadingRemarksUrlBase       = configuration.get[String]("urls.declareTransitMovementUnloadingFrontend")
-  val declareUnloadingRemarksUrl: String => String = mrn => s"$declareUnloadingRemarksUrlBase/$mrn/unloading-guidance"
-  val declareArrivalNotificationUrl: String        = configuration.get[String]("urls.declareTransitMovementArrivalFrontend")
+  private val declareUnloadingRemarksUrlBase        = configuration.get[String]("urls.declareTransitMovementUnloadingFrontend")
+  val declareUnloadingRemarksUrl: String => String  = mrn => s"$declareUnloadingRemarksUrlBase/$mrn/unloading-guidance"
+  private val declareArrivalNotificationUrl: String = configuration.get[String]("urls.declareTransitMovementArrivalFrontend")
```

Comment:
Rename this to include Base. Putting the two base urls together in this file might be easier to read. I'll let you decide

Reply:
Done